docs/app/_components/SiteMenu.tsx

```diff
@@ -101,16 +302,72 @@ export const SiteMenu = () => {
                 {items.map(page => (
                   <Command.Item
                     className={classNames.item}
-                    key={page.slug}
-                    value={page.slug}
-                    onSelect={() => goto(page.slug)}
+                    key={page.href}
+                    value={page.href}
+                    onSelect={() => window.open(page.href, '_blank')}
+                    keywords={page.keywords}
                   >
-                    {page.title}
+                    <Inline space={4} alignY="center">
+                      {page.name}
+                      <Split />
+                      <span className="text-text-primary-muted text-xs">
+                        <ExternalLink />
+                      </span>
+                    </Inline>
                   </Command.Item>
                 ))}
               </CommandGroup>
             ))}
+            {/* tokens copy command */}
+            {query && (
+              <CommandGroup
+                heading="Colors"
+                key="color"
+                className={classNames.section}
+              >
+                {tokens.map(([token]) => (
+                  <CopyItem
+                    className={classNames.item}
+                    key={token}
+                    value={token}
+                    keywords={['copy']}
+                    copyValue={token.replace('-DEFAULT', '')}
+                  >
+                    {token.replace('-DEFAULT', '')}
+                  </CopyItem>
+                ))}
+              </CommandGroup>
+            )}
+            {/* copy icon command */}
+            {query && (
+              <CommandGroup
+                heading="Icons"
+                key="icons"
+                className={classNames.section}
+              >
+                {Object.values(iconElements).map(elements => (
+                  <CopyItem
+                    key={elements.icon}
+                    value={elements.icon}
+                    keywords={['copy']}
+                    className={classNames.item}
+                    copyValue={elements.svg}
+                  >
+                    {elements.iconElement}
+                    {elements.icon}
+                  </CopyItem>
+                ))}
+              </CommandGroup>
+            )}
           </Command.List>
+          <div className="flex h-10 items-center justify-end gap-4 border-t px-2">
+            <div>
+              Action <span className="opacity-50">↵</span>
+            </div>
+            <div>
+              More Details <Hotkey letter="D" />
```

Comment:
Maybe this would look better if the hotkey would look more like a anctually key in this scenario?

Reply:
what do you mean?

Reply:
the hotkey is currently only for command key (hard coded) and than a letter you want (currently K and D)

Reply:
I meant the styling here. in the navigation I think its fine, but in the menu, maybe it should be more prominent. 

e.g. make it look like an actual keyboard key: https://codepen.io/dudleystorey/pen/AgEpXj